checker/tests/run-pass/test_source.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+#[macro_use]
+extern crate mirai_annotations;
+
+//=TEST_ONLY
+//=NO_ANGELIC
+
+#[test]
+fn some_test() {
+    verify!(1 == 1);
```

Comment:
Test methods are likely to use assert!, or in the case where the programmer is MIRAI aware, checked_verify! After all, we expect that these tests will have interesting runtime behavior.